components/analytics/compact-dashboard.tsx

```
@@ -0,0 +1,781 @@
+                                                            </TableCell>
+                                                            <TableCell className="text-right font-medium">{group.count}</TableCell>
+                                                        </TableRow>
+                                                    ));
+                                                })()}
+                                            </TableBody>
+                                        </Table>
+                                    </div>
+                                </CardContent>
+                            </Card>
+
+                            {data && data.errorsByDate && data.errorsByDate.length > 0 && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)